backend/fms_core/viewsets/_fetch_data.py

```diff
@@ -105,19 +101,13 @@ def fetch_data(self, ids: List[int] =[]) -> Tuple[List, int]:
             'quantity_flag',
             'depleted',
             'comment',
-            'count_derived_samples',
-            'first_derived_sample',
-            'first_project_id',
             'created_by',
             'created_at',
             'updated_by',
             'updated_at',
             'deleted',
-            'count_derived_by_sample',
         )
 
-        count = self.queryset.count() # Get count after value to have rows merged but before paging to have complete count
-
         if self.fetch_limit is not None and self.fetch_offset is not None:
             self.queryset = self.queryset[self.fetch_offset:self.fetch_offset+self.fetch_limit] # page the queryset
```

Comment:
Why not split this into two if statement?

```python
self.queryset = self.queryset[self.fetch_offset:]
self.queryset = self.queryset[:self.fetch_limit]
```

Reply:
This is not equivalent...

Reply:
I know it's not equivalent. But, maybe the user would like to paginate with only either offset or limit?

Reply:
This is a old bit of code unrelated to the current issue ... we should talk this out and create an independent ticket for that. I think what you said makes sense but i do not know why we did it that way in the first place ...